content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -28,6 +28,12 @@ To install and configure Istio in a Kubernetes cluster, follow these instruction
 
 1. Check the [Requirements for Pods and Services](/docs/setup/kubernetes/spec-requirements/).
 
+## Production install
+
+For a production setup of Istio, we recommend installing with the
```

Comment:
If you want to warn users installing on production environments, please use a warning callout: https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts
The warning would be best placed just before the Prerequisites heading.